web/src/app/services/user.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { AngularFireAuth } from 'angularfire2/auth';
+import { from } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+import {
+  AngularFirestore,
+  AngularFirestoreCollection
+} from 'angularfire2/firestore';
+import { ErrorService } from './error.service';
+
+@Injectable()
+export class UserService {
+  public user: AngularFirestoreCollection;
+  public userSocial: AngularFirestoreCollection;
+  constructor(
```

Comment:
Space needed between class variables and constructor

Reply:
Okay